paasta_tools/metrics/metastatus_lib.py

```diff
@@ -550,12 +574,13 @@ def get_marathon_status(client):
     )
 
 
-def assert_chronos_scheduled_jobs(client):
+def assert_chronos_scheduled_jobs(client, can_fail=False):
```

Comment:
is there any reason why we shouldn't have 0 jobs? I say ditch the can_fail - if you don't want to check it, then dont call the function